src/pages/ChemicalDetails.vue

```diff
@@ -35,15 +41,27 @@ export default {
 
   },
   computed: {
+    
+    chemData: function() {
+      var lookup = this.chemName;
+      var items = this.$store.getters.chemicals;
+      var result = [];
+      for(var item, i=0; item=items[i++];) {
+        if ((item.chemical_name).toLowerCase() == lookup.toLowerCase()) {
```

Comment:
function is O(n^2), just mark it as slow w/ a comment - if we ever find ourselves having slow chemical lookups, this could be a place to optimize.

Reply:
done